AccessCheckoutDemo/AccessCheckoutDemoUITests/pageObjects/CardFlowViewPageObject.swift

```diff
@@ -72,6 +72,17 @@ class CardFlowViewPageObject {
             panField.tap()
         }
         panField.typeText(text)
+
+    }
+    
+    func setCursorAndTypeInPanField(_ characters: [String]) {
```

Comment:
Can we rename this so that the function name includes where the cursor is set?

Reply:
updated